components/SectionsColumn.js

```diff
@@ -76,13 +79,19 @@ export const SectionsColumn = ({
     }
   }, [])
 
+  const updateSlugsOnAdd = (previousState, section) => {
+    return previousState.filter((slug) => slug !== section)
+  }
+
   const onAddSection = (e, section) => {
     localStorage.setItem('current-focused-slug', section)
     setpageRefreshed(false)
     setAddAction(true)
-    setSectionSlugs((prev) => prev.filter((s) => s !== section))
+    setSectionSlugs((prev) => updateSlugsOnAdd(prev, section))
+    setFilteredSlugs((prev) => updateSlugsOnAdd(prev, section))
     setSelectedSectionSlugs((prev) => [...prev, section])
     setFocusedSectionSlug(localStorage.getItem('current-focused-slug'))
+    resetSearchFilter('')
```

Comment:
```suggestion
    resetSearchFilter()
```

don't think you need that argument!

Reply:
Hi @octokatherine,

Nice catch 😄  I don't know how I missed this! I've pushed the requested changes.

Thank you 😊